packages/components/col/__tests__/layout.spec.tsx

```diff
@@ -31,15 +31,14 @@ describe('Col', () => {
   })
 
   it('gutter', () => {
-    const TestComponent = {
-      template: `<el-row :gutter="20">
-      <el-col :span="12" ref="col"></el-col>
-    </el-row>`,
-      components: {
-        'el-col': Col,
-        'el-row': Row,
-      },
-    }
+    const TestComponent = defineComponent({
+      setup: () => () =>
+        (
+          <Row gutter={20}>
+            <Col span={12} ref="col"></Col>
+          </Row>
+        ),
+    })
     const wrapper = mount(TestComponent)
```

Comment:
Can we do:

```tsx
mount(<Row gutter={20}>
            <Col span={12} ref="col" />
          </Row>)
```
here? Since we don't have any variables